packages/account/src/Components/currency-selector/currency-selector.jsx

```diff
@@ -102,7 +103,11 @@ const CurrencySelector = ({
     const description = is_dxtrade_allowed ? (
         <Localize i18n_default_text='You are limited to one fiat account. You won’t be able to change your account currency if you have already made your first deposit or created a real DMT5 or Deriv X account.' />
     ) : (
-        <Localize i18n_default_text='You are limited to one fiat account. You won’t be able to change your account currency if you have already made your first deposit or created a real DMT5 account.' />
+        <Localize
+            i18n_default_text={`You are limited to one fiat account. You won’t be able to change your account currency if you have already made your first deposit${
+                residence === 'im' ? '' : 'or created a real DMT5 account'
+            }.`}
+        />
```

Comment:
@mikita-deriv 
Here, `or created a real DMT5 account` won't get translated. Our translation extraction script won't pick this.

Instead, we can add a condition to check if `is_mt5_allowed` along with `is_dxtrade_allowed`  and based on it  we can set the description
```
const description = React.useMemo(() => {
    if (is_dxtrade_allowed && is_mt5_allowed) {
         return <Localize ...
    }
    else if (is_mt5_allowed){
      return <Localize ...
    }
    return <Localize ...

      
}, [is_dxtrade_allowed, is_mt5_allowed]);
```

Reply:
will fix it, thanks, interesting that we have this bug already in production according to this PR - https://github.com/binary-com/deriv-app/pull/2627